orc8r/tools/packer/magma-focal-virtualbox.json

```diff
@@ -75,7 +75,7 @@
         ]
       ],
       "virtualbox_version_file": ".vbox_version",
-      "vm_name": "magma"
+      "vm_name": "magma-dev"
```

Comment:
We see io degeneration with the created VM. The created VM uses an IDE controller - the image before had SCSI controller. Could this be a reason for the regression?
Can be set here via `"hard_drive_interface": "scsi"`